learn/what_is_meilisearch/features.md

```diff
@@ -18,51 +18,48 @@ Meilisearch's default [relevancy rules](/learn/core_concepts/relevancy.md) are d
 
 ## Sorting
 
-[Sort search results](/learn/advanced/sorting.md) at query time and let users choose which types of results they want to see first.
-
-## Placeholder search
+[Sort search results](/learn/advanced/sorting.md) at query time and let users choose which results they want to see first.
 
-If you make a search without inputting any query words, Meilisearch will return all the documents in that index sorted by its [custom ranking rules](/learn/core_concepts/relevancy.md#custom-rules) and [sorting rules](/learn/advanced/sorting.md#sorting). This feature is called **placeholder search**.
+## API key management
 
-Placeholder searches are particularly effective when used with other features such as [faceting or filtering](/learn/advanced/filtering_and_faceted_search.md#filters-or-facets), which allow users to narrow their searches and browse by category. You can read more about this feature in our article on [search parameters](/reference/api/search.md#placeholder-search).
+[Protect your instance with API keys](/learn/security/master_api_keys.md). Control which users can access what indexes, routes, and endpoints.
 
-## Phrase search
+## Multitenancy
 
-If you enclose search terms in double quotes (`"`), Meilisearch will only return documents that contain those terms in the order they were given. This gives users the option to make more precise search queries.
+Manage complex multi-user applications handling sensitive data. [Tenant tokens](/learn/security/tenant_tokens.md) help you decide which documents each one of your users can search.
 
-Phrase search is particularly useful when looking for a single result, such as searching for a book by ISBN. It is also possible to combine phrase searches with Meilisearch's basic syntax so only parts of a query are matched strictly.
+## Synonyms
 
-## API key management
+[Define synonyms](/learn/configuration/synonyms.md) and craft a tailored, intuitive search experience.
 
-Meilisearch allows you to protect your instances through the use of API keys. API keys give you control over which users can access what indexes, routes, and endpoints.
+## Highlighting
 
-API keys are managed by the master key. When you set a master key on your first launch, Meilisearch generates two API keys called the `Default Search API Key` and the `Default Admin API Key`. You can read more them in our article on [security](/learn/security/master_api_keys.md).
+[Highlight](/reference/api/search.md#attributes-to-highlight) query terms and help users immediately find the matched text in a document.
 
 ## Comprehensive language support
 
 [Meilisearch is multilingual](/learn/what_is_meilisearch/language.md)! We aim to support every language represented in our global community.
+
+## Phrase search
+
+[Wrap search terms in double quotes (`"`) for strict queries](/reference/api/search.md#phrase-search) that only return exact matches.
+
+## Index swapping
```

Comment:
Think this should be moved up a bit to follow tenant tokens, perhaps. Makes sense to end on language support IMO.